src/Icons/doc/index.rst

```diff
@@ -19,10 +19,60 @@ Installation
 
     $ composer require symfony/ux-icons
 
-Icons?
+Icons
 ------
 
-No icons are provided by this package but there are several ways to include and render icons.
+The ``symfony/ux-icons`` package aspires to offer simple and easy-to-use ways to
+handle SVG icons in your Symfony application.
+
+SVG Icons
+~~~~~~~~~
+
+[SVG](https://en.wikipedia.org/wiki/SVG) (Scalable Vector Graphics) is an XML-based
+vector image format. SVGs are a great way to add scalable, resolution-independent
```

Comment:
I think we should tone-down this paragraph. Let's remove expressions like `a great way`, `perfect choice`, etc.

SVG icons have many advantages and we should mention them, but let's do that in a more neutral way. Thanks!

Reply:
I may have been a bit over-enthusiast :)